src/components/common/ConsumptionCard.tsx

```diff
@@ -0,0 +1,67 @@
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            width: '90%',
+            marginTop: theme.spacing(2),
+            [theme.breakpoints.down('sm')]: {
+                width: `calc(100% - ${theme.spacing(1)}px)`,
+                margin: theme.spacing(1),
+            },
+        },
+        bullet: {
+            display: 'inline-block',
+            margin: '0 2px',
+            transform: 'scale(0.8)',
+        },
+        title: {
+            fontSize: 14,
+        },
+        pos: {
+            marginBottom: 12,
+        },
+    }),
+);
+
+export default function DetailsCard(props: {
+    peakValue: number;
+    averageValue: number;
+    comparedToAverage: number;
+    totalCost: number;
+    totalConsumption: number;
+}): React.ReactElement {
+    const classes = useStyles();
+    const { formatMessage } = useIntl();
+    const { peakValue, averageValue, comparedToAverage, totalCost, totalConsumption } = props;
+
+    return (
+        <Card className={classes.root} variant="outlined">
+            <CardContent>
+                <Typography className={classes.title} color="textSecondary" gutterBottom>
+                    {formatMessage({ id: 'Consumption Summary' })}
```

Comment:
it feels a bit weird to have ids with empty spaces and not came case.
I'm not sure how well this works with different characters, but I would use json format with .